processor/groupbytraceprocessor/documentation.md

```diff
@@ -6,66 +6,66 @@
 
 The following telemetry is emitted by this component.
 
-### otelcol_processor_groupbytrace_conf_num_traces
+### processor_groupbytrace_conf_num_traces
```

Comment:
do you know what happened here?

Reply:
I ran ```go generate ./... ```  and it was updated automatically. I ran this to make sure documentation is up to date with metadata.yaml

Reply:
@codeboten, do you know if this is something concerning?